sheet/zedestructor/Anya.qml

```diff
@@ -378,6 +380,24 @@ Item {
         }
     }
 
+    SavingThrow {
+        name: "Fortitude"
+        ability: constitution
+        primary: false
+    }
+
+    SavingThrow {
+        name: "Reflex"
+        ability: dexterity
+        primary: true
+    }
+
+    SavingThrow {
+        name: "Willpower"
```

Comment:
I think this should just be called 'Will', like it does on the official character sheet.